osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -8,31 +10,37 @@
 using osu.Game.Beatmaps;
 using osu.Game.Database;
 using osu.Framework.Graphics.Colour;
+using osu.Game.Beatmaps.Drawables;
 
 namespace osu.Game.Screens.Select
 {
-    class BeatmapInfoOverlay : Container
+    class BeatmapInfoWedge : Container
     {
         private Container beatmapInfoContainer;
 
+        private BaseGame game;
+
+        [BackgroundDependencyLoader]
+        private void load(BaseGame game)
+        {
+            this.game = game;
+        }
+
         public void UpdateBeatmap(WorkingBeatmap beatmap)
         {
             if (beatmap == null)
                 return;
 
-            float newDepth = 0;
-            if (beatmapInfoContainer != null)
-            {
-                newDepth = beatmapInfoContainer.Depth - 1;
-                beatmapInfoContainer.FadeOut(250);
-                beatmapInfoContainer.Expire();
-            }
+            var lastContainer = beatmapInfoContainer;
+
+            float newDepth = lastContainer?.Depth - 1 ?? 0;

```

Comment:
I just realized we are in-fact fading in below, and not keeping the previous container to fade out. You should be able to remove `newDepth` altogether.